src/main/java/com/azure/cosmos/examples/modeling/async/Deployment.java

```diff
@@ -0,0 +1,310 @@
+package com.azure.cosmos.examples.modeling.async;
```

Comment:
Do we need copyright? I saw some of the files have, some do not

Reply:
we do need copyright